src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -252,6 +252,7 @@ When running `benchmark latency`, two types of tests will be executed:
 |:------------------------------------|:----------------------------------------------------------------------------|
 | `--intra-cluster`                   | Performs the benchmark test within a single cluster between Pods from a Non-Gateway node to a Gateway node
 | `--verbose`                         | Produce verbose logs during benchmark tests
+| `--kubeconfig` `<string>`           | Absolute path(s) to the kubeconfig file(s)
```

Comment:
It seems we should also document `--kubecontexts  | comma-separated list of two kubeconfig contexts to use.`

Reply:
I followed the same pattern as in `subctl verify`, in both cases using `--kubecotexts` is not optional and documented as part of the command usage

Reply:
Ok. For `verify`, `--kubeconfig` is not documented so perhaps we shouldn't here as well for consistency?